test/utils/introspection/SupportsInterface.behavior.js

```diff
@@ -92,7 +92,7 @@ const SIGNATURES = {
 
 const INTERFACE_IDS = mapValues(SIGNATURES, interfaceId);
 
-function shouldSupportInterfaces(interfaces = []) {
+function shouldSupportInterfaces(interfaces = [], signatures = SIGNATURES, interfaceIds = INTERFACE_IDS) {
```

Comment:
This allows inconsistent parameters. IMO we should only take the "signatures", and recompute the "interfaceIds" in the function.

Reply:
```suggestion
function shouldSupportInterfaces(interfaces = [], signatures = SIGNATURES) {
    const interfaceIds = mapValues(SIGNATURES, interfaceId);
```

Reply:
This make sense and it doesn't increase the test time considerably